task_1_1_1__/HeapSort.java

```diff
@@ -0,0 +1,52 @@
+
+import java.util.*;
+import java.util.Arrays;
+
+public class HeapSort {
```

Comment:
You should add gradle files for application assembling and starting

(The easiest way is to re-create project within IDEA as Gradle project and to move your source files to it)

Also, you should create .sh or .bat script for compiling, generating javadocs and starting your programm (it is additinal task it task 1).
You can organize it as single script, which runs specific action depending on arguments (run keys), or by creating separate file for each action
For compile see `javac`, for run `java`, for generating documentation - `javadoc`